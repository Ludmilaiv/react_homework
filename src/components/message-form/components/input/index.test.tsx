import { Input } from './';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

describe('Input', () => {
  it('render component', () => {
    render(<Input autofocus={false} placeholder="text" value="" />);
  });

  it('render with snapshot', () => {
    const { asFragment } = render(
      <Input autofocus={false} placeholder="text" value="" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
