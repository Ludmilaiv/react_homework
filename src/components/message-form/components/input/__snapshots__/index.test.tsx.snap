// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input render with snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
  >
    <label
      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
      data-shrink="false"
      for="outlined-basic"
      id="outlined-basic-label"
    >
      text
    </label>
    <div
      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
    >
      <input
        aria-invalid="false"
        class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
        id="outlined-basic"
        type="text"
        value=""
      />
      <fieldset
        aria-hidden="true"
        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="css-1ftyaf0"
        >
          <span>
            text
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</DocumentFragment>
`;
